CleverTapSDK/InApps/CustomTemplates/CTCustomTemplate-Internal.h

```diff
@@ -23,7 +24,8 @@
                         templateType:(NSString *)templateType
```

Comment:
Keep the default constructor and add the one with `isSystemDefined` as an additional one. Reuse them. Revert the CustomTemplate Test changes.

Reply:
The constructor is used at only one place in actual code in `CTCustomTemplate` build method and this is internal method, and `isSystemDefined` can have both values in `CTCustomTemplate`. 
By keeping default constructor, it will only used in CustomTemplate test changes. So I have made only one constructor with `isSystemDefined`. Let me know if I am thinking correct or changes is needed.

Reply:
ok, this is fine